### PR TITLE
add dotnet version param do docker composes

### PR DESCRIPTION
**Pull request by pedromsmoreira**
Allow choosing DotNet image version in order to be more flexible.

TODO:
 Need to think of a default when the version is not defined in jenkinsfile

### CODE REVIEW

File: resources/docker-compose.unittests.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   ci-unit-tests:
-    image: microsoft/dotnet:1.1.2-sdk
+    image: microsoft/dotnet:${DOTNET_VERSION}
```
Comment (joao-r-reis) [inline resources/docker-compose.unittests.yml]: Maybe we should have a new variable for the SDK version?

**Review by joao-r-reis — CHANGES_REQUESTED** [review]
we probably need a different variable for the sdk, or maybe we could do {dotnet_version}-sdk? I'm not sure if the versions for the different dotnet images match.

File: resources/docker-compose.unittests.yml (hunk 1 above)
Comment (pedromsmoreira) [reply resources/docker-compose.unittests.yml]: Yes, we should have one. I will create and add it to WithEnv method.

**Comment by pedromsmoreira** [comment]
When I changed the versions in docker compose, both versions were aligned. I will re-check in Microsoft Docker Hub.

**Comment by pedromsmoreira** [comment]
I adapted to ${DOTNET_SDK_VERSION}-sdk

It is clearer to understand. Although we have to indicate both aspnetcore and dotnet-sdk versions. (With this comment I will change the var name to aspnetcoreVersion)

**Pull request merged**